### PR TITLE
Fix dealing with udevadm --version reporting sth unexpected

### PR DESCRIPTION
**Pull request by mikhailnov**
Fixes https://github.com/dracutdevs/dracut/issues/668

### CODE REVIEW

File: dracut-init.sh [hunk 1]
```diff
@@ -461,6 +461,14 @@ inst_rules_wildcard() {
 
 prepare_udev_rules() {
     [ -z "$UDEVVERSION" ] && export UDEVVERSION=$(udevadm --version)
+    if [ -z "$UDEVVERSION" ]; then
+        derror "Failed to detect udev version"
```
Comment (danimo) [inline dracut-init.sh]: Punctuation. Maybe Add '!' (see below)?

File: dracut-init.sh [hunk 2]
```diff
@@ -461,6 +461,14 @@ inst_rules_wildcard() {
...
+        return 1
+    fi
+    if [ -z "${UDEVVERSION##*[!0-9]*}" ]; then
+        derror "udevadm --version reported not an integer, can't handle it"
```
Comment (danimo) [inline dracut-init.sh]: Spelling: "...did not report an integer, ...".  can't -> cannot.

And "can't handle it is a bit implicit with derror, isn't it? ;-) " Maybe better:

"udevadm --version did not report an integer, udev version cannot be determined!"

(unfortunately, dracut is not consistent when it comes to punctuation here)

Comment (mikhailnov) [reply dracut-init.sh]: Done, force-pushed.

**Comment by mikhailnov** [comment]
@danimo , applied what you suggested and force-pushed

**Pull request merged**